cmd/maya-apiserver/spc-watcher/handler.go

```diff
@@ -139,3 +147,40 @@ func (c *Controller) getSpcResource(key string) (*apis.StoragePoolClaim, error)
 	}
 	return spcGot, nil
 }
+
+func syncSpc(spcGot *apis.StoragePoolClaim) error {
+	if len(spcGot.Spec.Disks.DiskList) > 0 {
+		// TODO : reconciliation for manual storagepool provisioning
+		return fmt.Errorf("no reconciliation for manual provisioning for storagepoolclaim %s", spcGot.Name)
+	}
+	glog.Infof("Syncing storagepoolclaim %s", spcGot.Name)
```

Comment:
Will this log flood the maya-api-server logs?

Reply:
Yes it will. Every time a resync happens.

Reply:
Can we make it a debug log then?

Reply:
Sure

Reply:
fixed